runtime/src/block_cost_limits.rs

```diff
@@ -56,3 +56,6 @@ pub const MAX_BLOCK_UNITS: u64 =
 /// limit is to prevent too many transactions write to same account, threrefore
 /// reduce block's paralellism.
 pub const MAX_WRITABLE_ACCOUNT_UNITS: u64 = MAX_BLOCK_REPLAY_TIME_US * COMPUTE_UNIT_TO_US_RATIO;
+
+/// max len of account data in a slot (bytes)
+pub const MAX_ACCOUNT_DATA_LEN: u64 = 100_000_000;
```

Comment:
If max account data len is 100 MB, then getting to 1 TB would only take a bit over 1 hour, is that right? I don't know what the right number should be here though... If a DoS was happening, would 1 hour be long enough to prevent/handle it?

Math used: `10^12 B / 10^8 B * 0.4 s per slot / 60 s per min / 60 min per hour` (double check in case I made dumb mistakes)

Reply:
@carllin ran these calculations in his head about 15 times last night.

Reply:
this comment was my invitation for him to comment ;-)

Reply:
Yeah about an hour. I think if we want to survive longer, we'll have to require validators have a lot more disk. Maybe 10TB minimum

Reply:
We would want more like a week. How does this max compare with the current mainnet-beta account data average written per slot?

Reply:
I've added the data len to the metrics. I'm waiting on the ability to start a validator with this to get the metrics of mnb. Keep in mind the current metrics ONLY measure the data size of account creation; not updates.